lib/livebook_web/live/session_live/secrets_list_component.ex

```diff
@@ -144,28 +148,19 @@ defmodule LivebookWeb.SessionLive.SecretsListComponent do
               <.hidden_field field={f[:name]} value={@secret.name} />
             </.form>
           </div>
-          <div class="flex flex-row justify-between items-center my-1 hidden" id={"#{@id}-detail"}>
-            <span class="text-sm font-mono break-all flex-row">
-              <%= Session.Data.secret_used_value(@secret, @secrets) %>
+          <div class="flex-row justify-between items-center my-1 hidden" id={"#{@id}-detail"}>
+            <span class="text-sm font-mono font-bold break-all flex-row">
```

Comment:
Let's not call attention to something that portrays no information.

```suggestion
            <span class="text-sm font-mono break-all flex-row">
```